dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/NettyChain.java

```diff
@@ -131,155 +135,155 @@ public synchronized void stop() {
                 tcpOptions.put(ConfigConstants.EXTERNAL_NAME, endpointName);
 
+                
+
                 bootstrap = nettyFramework.createTCPBootstrap(tcpOptions);
                 HttpPipelineInitializer httpPipeline = new HttpPipelineInitializer.HttpPipelineBuilder(this)
                                 .with(ConfigElement.COMPRESSION, owner.getCompressionConfig())
-                                .with(ConfigElement.HTTP_OPTIONS, httpOptions)
-                                .with(ConfigElement.HEADERS, owner.getHeadersConfig())
+                                .with(ConfigElement.HTTP_OPTIONS,httpOptions)
+                                .with(ConfigElement.HEADERS,owner.getHeadersConfig())
                                 .with(ConfigElement.REMOTE_IP,owner.getRemoteIpConfig())
-                                .with(ConfigElement.SAMESITE, owner.getSamesiteConfig()).build();
+                                .with(ConfigElement.SAMESITE,owner.getSamesiteConfig())
+                                .build();
 
                 bootstrap.childHandler(httpPipeline);
 
                 serverChannel = nettyFramework.start(bootstrap, info.getHost(), info.getPort(), this::channelFutureHandler);
 
+
                 VirtualHostMap.notifyStarted(owner, () -> currentConfig.getResolvedHost(), currentConfig.getConfigPort(), isHttps);
                 String topic = owner.getEventTopic() + HttpServiceConstants.ENDPOINT_STARTED;
                 postEvent(topic, currentConfig, null);
-                
-                
 
             } catch (Exception e) {
-                MSP.log("Failed to start NettyChannel: " + e.getMessage());
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.exit(this, tc, "Failed to start Netty Channel: " + e.getMessage());
+                }
                 state.set(ChainState.STOPPED);
-            } finally {
-                notifyAll();
             }
         }
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.exit(this, tc, "Finished starting Netty Channel: " + endpointName + ", Final state: " + state.get());
+        }
 
     }
 
     private void channelFutureHandler(ChannelFuture future) {
-        if (future.isSuccess()) {
-            state.set(ChainState.STARTED);
-            EndPointInfo info = endpointMgr.getEndPoint(this.endpointName);
-            info = endpointMgr.defineEndPoint(this.endpointName, currentConfig.configHost, currentConfig.configPort);
-            // VirtualHostMap.notifyStarted(owner, () -> currentConfig.getResolvedHost(), currentConfig.getConfigPort(), isHttps);
-            // String topic = owner.getEventTopic() + HttpServiceConstants.ENDPOINT_STARTED;
-            // postEvent(topic, currentConfig, null);
 
-            MSP.log("Channel is now active and listening on port " + getActivePort());
+        synchronized(this) {
```

Comment:
Maybe this sync isn't exactly what we expect given that this runs in the future and this could no longer be the class